pkg/controller/windowsmachineconfig/nodeconfig/nodeconfig.go

```diff
@@ -209,39 +232,10 @@ func (nc *nodeConfig) handleCSR(requestorFilter string) error {
 	return nil
 }
 
-// GetNode returns a pointer to the node object associated with the external IP provided
-func (nc *nodeConfig) getNode(externalIP string) (*v1.Node, error) {
-	var matchedNode *v1.Node
-
-	nodes, err := nc.k8sclientset.CoreV1().Nodes().List(metav1.ListOptions{})
-	if err != nil {
-		return nil, errors.Wrap(err, "could not get list of nodes")
-	}
-	if len(nodes.Items) == 0 {
-		return nil, fmt.Errorf("no nodes found")
-	}
-
-	// Find the node that has the given IP
-	for _, node := range nodes.Items {
-		for _, address := range node.Status.Addresses {
-			if address.Type == "ExternalIP" && address.Address == externalIP {
-				matchedNode = &node
-				break
-			}
-		}
-		if matchedNode != nil {
-			break
-		}
-	}
-	if matchedNode == nil {
-		return nil, fmt.Errorf("could not find node with IP: %s", externalIP)
-	}
-	return matchedNode, nil
-}
-
```

Comment:
Please make the `getNode` changes here.

Reply:
wrt https://github.com/openshift/windows-machine-config-operator/pull/36#discussion_r408389472

Reply:
was misled by https://github.com/openshift/windows-machine-config-operator/pull/36#discussion_r408389472